Bump eipv to v0.0.5

lightclient opened the pull request:
The validator was not correctly checking email addresses. This has been fixed in `eipv v0.0.5`: https://github.com/lightclient/eipv/pull/8

MicahZoltu commented:
@axic I think this resolves the issue you mentioned in a PR?  @lightclient It appears CI correctly located the malformed email address and that needs to be fixed for this to merge.  Ideally, the author of that EIP would fix it, but to avoid having to wait indefinitely for that consider adding a fix to this PR (just change to `()` from `<>` since I think that is what the author meant anyway).

lightclient commented:
@MicahZoltu yes, you beat me to it. I went ahead and fixed it here.

@axic, thank you for point this error out. It appears `eipv` did correctly run on that PR, but it did not check the validity of the email. I have now fixed it.

eip-automerger commented:
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - File .travis.yml is not an EIP
 - EIP 2746 requires approval from one of (@juanfranblanco, @jaerith)
